ios/LayoutReanimation/REASharedTransitionManager.h

```diff
@@ -0,0 +1,19 @@
+#import <RNReanimated/REAAnimationsManager.h>
+#import <RNReanimated/REASnapshot.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface REASharedTransitionManager : NSObject
+
+- (void)notifyAboutNewView:(UIView *)view;
+- (void)viewsDidLayout;
+- (BOOL)setupSyncSharedTransitionForViews:(NSArray<UIView *> *)views;
+- (void)finishSharedAnimation:(UIView *)view;
+- (NSArray<UIView *> *)getCurrentSharedTransitionViews;
```

Comment:
I think `getCurrentSharedTransitionViews` is neither implemented nor used anywhere (please correct me if I'm wrong)